docs/test/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+####################################################################################################
+# Add tests
+####################################################################################################
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/grep.py
+    ${CMAKE_CURRENT_BINARY_DIR}/grep.py COPYONLY)
+
+find_package(Python COMPONENTS Interpreter Development)
+if(NOT(${Python_FOUND} AND (${Python_VERSION} VERSION_GREATER "3.6")))
+    message(FATAL_ERROR "Building the documentation tests requires Python 3.7")
```

Comment:
Shouldn't this assert or finish test execution?

Reply:
This is coming from other projects :sweat_smile: But looking at [CMake documentation](https://cmake.org/cmake/help/v3.0/command/message.html) when the `mode` is `FATAL_ERROR`: `CMake Error, stop processing and generation`. So it seems like it is already doing what you suggest. :wink: